dashboard/app/models/unit_group.rb

```diff
@@ -406,6 +362,10 @@ def summarize_short
     }
   end
 
+  def included_in_units?(unit_ids)
```

Comment:
(minor) Does a unit group contain units or vice versa?  (If unit groups contain units, maybe `contains_units?` would be a clearer name?)

Reply:
Ya I didn't love this name for unit_group but I wanted to use the same name for the unit_group method as the script method so that course_version did not need special casing. I was having a hard time thinking of a better name. The method used to be 'has_student_progress' but when I tried to make things more generic I landed here. I'm open to suggestions of something that would work for both unit and unit_group

Reply:
I guess it could be contains_units? a unit kind of contains itself